### ISSUE TITLE
Simplified Chinese Translation

### ISSUE THREAD

**Issue by soleo** [GoogleChrome/essential-image-optimization#57]
Hi, there,

I'm interested in translating this repo to a simplified Chinese.

**Comment by hkongm** [comment]
这么好的文章，是应该有中文版！！！

**Comment by jhabdas** [comment]
Given the L10n/i18n requests are popping up probably a good course of action for those interested would be to take some inspiration for the design of the Yarn website and how they handle translations using contributor-based crowdsourcing:

https://github.com/yarnpkg/website/

**Comment by soleo** [comment]
@jhabdas good idea! I checked out the yarn website, and it looks that their website is using the vendor (https://crowdin.com), and they would offer free price for open source projects. I did configuration for my fork to start translation(https://crowdin.com/project/essential-image-optimization), and it looks good so far. @addyosmani are you up for it?

**Comment by soleo** [comment]
FYI, the PR in my fork is https://github.com/soleo/essential-image-optimization/pull/1

**Comment by soleo** [comment]
If anyone is interested in language translation, join via https://crwd.in/essential-image-optimization

**Comment by ProteanBear** [comment]
I translated this book into Simplified Chinese.
You can see in my fork [ProteanBear](https://github.com/ProteanBear/essential-image-optimization/tree/translation_cn)
